### PR TITLE
sql: test some more simplifications

### PR DESCRIPTION
**Pull request by andreimatei**
I was playing around with simplifications; I thought this is worth committing because these tests are useful for documenting current behavior.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3463)

### CODE REVIEW

**Comment by tamird** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3463)*

File: sql/analyze_test.go [hunk 1]
```diff
@@ -208,6 +209,9 @@ func TestSimplifyExpr(t *testing.T) {
 		{`(a, b) = (1, 2)`, `(a, b) IN ((1, 2))`, true},
 		{`(a, b) = (1, 2) OR (a, b) = (3, 4)`, `(a, b) IN ((1, 2), (3, 4))`, true},
 		{`(a, b) IN ((2, 1), (1, 2), (1, 2), (2, 1))`, `(a, b) IN ((1, 2), (2, 1))`, true},
+		// Expressions that don't simplify properly:
```
Comment (petermattis) [inline sql/analyze_test.go]: I assume this really means that the expressions don't simplify when they could. That is, the current behavior isn't erroneous, just not optimal.

**Comment by petermattis** [comment]
LGTM

File: sql/analyze_test.go (hunk 1 above)
Comment (andreimatei) [inline sql/analyze_test.go]: right. I've clarified in the comment.

**Pull request merged**